src/frontend/src/components/irregularity.ts

```diff
@@ -0,0 +1,48 @@
+import { html, TemplateResult } from "lit-html";
+import { warningIcon, closeIcon } from "./icons";
+
+interface irregularityProps {
+  message: string | TemplateResult;
+  additionalClasses?: string[];
+  type?: "warning" | "info" | "error";
+  closeFn?: () => void;
```

Comment:
why is this optional? what happens if no `closeFn` is specified?

Reply:
@nmattia because I can see some cases where there should not be a close button. Sometimes they might be timed notification. Or an error that we want to show until the issue is resolved.